src/sspi/kerberos.rs

```diff
@@ -0,0 +1,598 @@
+mod client;
+pub mod config;
+mod encryption_params;
+pub mod network_client;
+mod server;
+mod utils;
+
+use std::{fmt::Debug, io::Write};
+
+use kerberos_crypto::new_kerberos_cipher;
+use lazy_static::lazy_static;
+use picky_krb::{
+    constants::key_usages::ACCEPTOR_SIGN,
+    data_types::KrbResult,
+    gss_api::{NegTokenTarg1, WrapToken},
+    messages::{ApRep, ApReq, AsRep, TgsRep},
+};
+use rand::{rngs::OsRng, Rng};
+
+use self::{
+    client::{
+        extractors::{
+            extract_encryption_params_from_as_rep, extract_session_key_from_as_rep,
+            extract_session_key_from_tgs_rep,
+        },
+        generators::{
+            generate_ap_req, generate_as_req, generate_authenticator_for_ap_req,
+            generate_authenticator_for_tgs_ap_req, generate_neg_ap_req, generate_neg_token_init,
+            generate_tgs_req,
+        },
+        AES128_CTS_HMAC_SHA1_96, AES256_CTS_HMAC_SHA1_96,
+    },
+    config::{KdcType, KerberosConfig},
+    encryption_params::EncryptionParams,
+    server::extractors::extract_tgt_ticket,
+    utils::{serialize_message, utf16_bytes_to_utf8_string},
+};
```

Comment:
Could you check [IronRDP’s rustfmt.toml](https://github.com/Devolutions/IronRDP/blob/master/rustfmt.toml) and run `cargo +nightly fmt` with the same configuration, please?

I think the only issue with `group_imports = "StdExternalCrate"` is the way it happily mixes `pub use`s with regular `use`s

Reply:
I ran `cargo fmt`. Also, I put the rustfmt.toml from `IronRDP` into the project.

Reply:
Perfect!